lib/resty/etcd/v3.lua

```diff
@@ -643,10 +643,11 @@ local function request_chunk(self, endpoint, method, scheme, host, port, path, o
         body, err = decode_json(body)
         if not body then
             return nil, "failed to decode json body: " .. (err or " unkwon")
-        elseif body.error and body.error.http_code >= 500 then
+        elseif body.error and body.error.http_code >= 400 then
```

Comment:
change the health_check threshold here, since error `grpc: the client connection is closing` (HTTP code 408) sometimes happens when etcd disconnected